pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -575,7 +575,7 @@ func (c *Controller) addNodeRoute(nodeName string, node *corev1.Node) error {
 		if peerNodeIP == nil {
 			peerNodeIP = peerNodeIPs.IPv6
 		}
-		if err := c.wireGuardClient.UpdatePeer(nodeName, peerWireGuardPublicKey, peerNodeIP, peerPodCIDRs); err != nil {
+		if err := c.wireGuardClient.UpdatePeer(nodeName, peerWireGuardPublicKey, peerNodeIP, 0, peerPodCIDRs); err != nil {
```

Comment:
Removed. This parameter will support different clusters using different WG ports. But Jianjun suggested to me that we can assume all clusters use the same port. So it can be removed.